### ISSUE TITLE
Problem with dataio_prepare function using csv files 

### ISSUE THREAD

**Issue by AurelienMarchal** [speechbrain/speechbrain#753]
Hello to the SpeechBrain team ! Your toolkit is awesome by the way thank you for creating it! 
I recently created a post on stackoverflow [(link)](https://stackoverflow.com/questions/67508634/speechbrain-dataio-prepare-function-with-csv) which is pretty detailed on the problem I am facing.

To summarize the issue, I am trying to follow the ASRfromScratch.ipynb tutorial with the [Fluent Speech DataSet](https://fluent.ai/fluent-speech-commands-a-dataset-for-spoken-language-understanding-research/). I managed to complete the tokenizer phase and the LM phase but I have trouble adapting the code of the Speech Recognition phase to the [Fluent Speech DataSet](https://fluent.ai/fluent-speech-commands-a-dataset-for-spoken-language-understanding-research/). I explained everything in the stackoverflow [post](https://stackoverflow.com/questions/67508634/speechbrain-dataio-prepare-function-with-csv).

It would be awesome if you could take a look at it and reply on the post or directly here! Thank you very much in advance !

**Comment by TParcollet** [comment]
Hi, you obtain this error because some of your data are stereo (or more) which is not managed by the padding this in this way. You will need to add: 

```python

info = torchaudio.info(wav)
        sig = sb.dataio.dataio.read_audio(wav)
        if info.num_channels > 1:
            sig = torch.mean(sig, dim=1)
        resampled = torchaudio.transforms.Re

```

Something like that. You also need to make the pretrainer work if you have a pretrained LM and Tokenizer, otherwise they won't be loaded ;)
